espnet/nets/pytorch_backend/e2e_asr.py

```diff
@@ -388,6 +454,67 @@ def subsample_frames(self, x):
         return h, ilen
 
 
+def _to_torch_tensor(x):
```

Comment:
This function may be moved to `nets_utils.py`.

Reply:
Moved.